src/one_sdk.spec.ts

```diff
@@ -217,28 +217,32 @@ describe('one_sdk', () => {
     });
 
     describe('provider configurations', () => {
-      it('sets empty object if parameters are not set', async () => {
```

Comment:
question: Why did you remove the test for `if parameters are not set` case? The bug is not covered by the tests now.

Reply:
I was thinking the same but planned sending a separate PR to add the test case.

Reply:
Ok I added it.